eng/Version.Details.xml

```diff
@@ -9,23 +9,41 @@
       <Sha>bdc031f53b9d24b5f3229badbb1d0b3b9e1e1fd9</Sha>
     </Dependency>
     <!-- Necessary for source-build. This allows the live version of the package to be used by source-build. -->
-    <Dependency Name="Microsoft.Build.Tasks.Core" Version="17.3.2">
+    <Dependency Name="Microsoft.Build.Tasks.Core" Version="17.7.2">
       <Uri>https://github.com/dotnet/msbuild</Uri>
-      <Sha>561848881bab01749e6d8b03be2869a18ca944f7</Sha>
+      <Sha>d6990bcfaf520c0d215a194fad0617f7efad68b4</Sha>
     </Dependency>
     <!-- Necessary for source-build. This allows the live version of the package to be used by source-build. -->
-    <Dependency Name="Microsoft.Build" Version="17.3.2">
+    <Dependency Name="Microsoft.Build" Version="17.7.2">
       <Uri>https://github.com/dotnet/msbuild</Uri>
-      <Sha>561848881bab01749e6d8b03be2869a18ca944f7</Sha>
+      <Sha>d6990bcfaf520c0d215a194fad0617f7efad68b4</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis" Version="4.9.0-3.24054.13">
+    <Dependency Name="Microsoft.Build.Framework" Version="17.7.2">
+      <Uri>https://github.com/dotnet/msbuild</Uri>
+      <Sha>d6990bcfaf520c0d215a194fad0617f7efad68b4</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Build.Tasks.Core" Version="17.7.2">
+      <Uri>https://github.com/dotnet/msbuild</Uri>
+      <Sha>d6990bcfaf520c0d215a194fad0617f7efad68b4</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Build.Utilities.Core" Version="17.7.2">
+      <Uri>https://github.com/dotnet/msbuild</Uri>
+      <Sha>d6990bcfaf520c0d215a194fad0617f7efad68b4</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.StringTools" Version="17.7.2">
+      <Uri>https://github.com/dotnet/msbuild</Uri>
+      <Sha>d6990bcfaf520c0d215a194fad0617f7efad68b4</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="3.2.2146">
+    </Dependency>
```

Comment:
This is missing info.

Reply:
project url added, but it doens't have any commit information in Nuget packages. (Seems like it is closed source repo)

Reply:
This shouldn't be added as a dependency then since it's not built by .NET.

Reply:
OK, removed